src/styles/blocks/header.scss

```diff
@@ -0,0 +1,67 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 100%;
+  padding: 0 50px;
+
+  &__logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+    object-fit: cover;
+  }
+}
+
+.nav-block {
+  display: flex;
+  align-items: center;
+  color: #000;
+
+  &__list {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 20px;
+    list-style: none;
+  }
+
+  &__item {
+    position: relative;
+  }
+
+  &__link {
+    position: relative;
+    display: inline-block;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 60px;
+    height: 60px;
+    text-transform: uppercase;
+    text-decoration: none;
```

Comment:
you wrote this rule `text-decoration: none;` twice

you can reset this style in your reset.scss, please add:
```
a {
  text-decoration: none;
}
```

after you added this rule don't forget do delete unnecessary styles from your scss blocks